examples/classic_controllers/classic_controllers_dc_motor_example.py

```diff
@@ -20,8 +20,8 @@
     """
 
     motor_type = 'PermExDc'
-    control_type = 'TC'
-    action_type = 'Cont'
+    control_type = 'SC'
+    action_type = 'F'
```

Comment:
Is this change just for more diversity in the examples or is there more to it?

Reply:
The changes in the examples were accidentially. I will redo them.